packages/itwinui-react/src/core/DropdownMenu/DropdownMenu.tsx

```diff
@@ -44,6 +44,7 @@ export type DropdownMenuProps = {
  *     Item #3
  *   </MenuItem>,
  * ];
+ *
```

Comment:
Nitpicky, but I would remove this added line just to keep the changes as concise as possible